content/blog/2024-07-31-tmix-07.md

```diff
@@ -0,0 +1,64 @@
++++
+title = "This Month in Xilem, July 2024"
+authors = ["Daniel McNab", "Olivier Faure"]
++++
+
+This month saw contributions on Linebender projects from volunteer contributors.
```

Comment:
I'm not sure that this sends out the right message, because we are still a community organisation, with open membership. Admittedly, the dynamics here are quite weird, due to our "personal" roadmaps and being full-time.

I do agree that Philipp's work has been excellent, and should be called out again. I'd suggest a sentence or two in the Xilem section for that.

(Bruce's work has also been extremely helpful, both in the visible changes being made and in performing reviews, and should be mentioned somewhere.)